lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -90,7 +89,7 @@ class CoursewareContextTestCase(ModuleStoreTestCase):
     comment client service integration
     """
     def setUp(self):
-        super(CoursewareContextTestCase, self).setUp()
+        super(CoursewareContextTestCase, self).setUp(create_user=True)
```

Comment:
it's weird that setUp() takes arguments like that, but it's nothing new and not from this PR right?